pkg/kubectl/service_basic.go

```diff
@@ -82,13 +83,27 @@ func (ServiceExternalNameGeneratorV1) ParamNames() []GeneratorParam {
 	}
 }
 
+// validatePortRange throws error if the given port is not vaild UDP or TCP port
+func validatePortRange(port int) error {
```

Comment:
nit: this could save one line, just my personal favor, your choice : )
```
func validatePortRange(port int) error {
	if port < 0 || port > 65535 {
		return errors.New("invalid port, out of range")
	}
	return nil
}
```